pages/template-page.js

```diff
@@ -1,18 +1,150 @@
+              Paste Docs
+            </PrototypeAnchor>
+            for a full list of components you can add.
+          </Paragraph>
+          <Paragraph>
+            To practice, let&apos;s add in a <strong>Separator</strong> component
+          </Paragraph>
+          <Paragraph>
+            To add a component to the &quot;About Me&quot; page:
+            <OrderedList>
+              <ListItem>
+                Find the import statements at the top of the <Code>about-me.js</Code> file.
+              </ListItem>
+              <ListItem>
+                Add the new component&apos;s import statement below the existing ones. For the{' '}
+                <strong>Separator</strong> component, this statement is{' '}
+                <Code>import &#123; Separator &#125; from &apos;@twilio-paste/core/separator&apos;;</Code>
+              </ListItem>
+              <ListItem>
+                Add the component itself into the code. For the <strong>Separator</strong> component, make a new line
+                after closing tag of the Heading component you edited in step 3. Then copy the following code and paste
+                it into the new line:{' '}
+                <Code>&lt;Separator orientation=&quot;horizontal&quot; verticalSpacing=&quot;space50&quot; /&gt;</Code>
+              </ListItem>
+            </OrderedList>
+            <Paragraph>
+              Once you save your work, a thin, horizontal separator line will appear beneath your heading!
+            </Paragraph>
+          </Paragraph>
+          <Disclosure>
+            <DisclosureHeading as="h4" variant="heading50">
+              More Details
+            </DisclosureHeading>
+            <DisclosureContent>
+              <UnorderedList>
+                <ListItem>
+                  You can adjust components by changing their properties, also called props. In the{' '}
+                  <strong>Separator</strong> component, verticalSpacing is a prop. Adjust it to &quot;space100&quot;
+                  instead of &quot;space50&quot; to see what happens.{' '}
+                  <PrototypeAnchor showExternal href="https://paste.twilio.design/components/separator">
+                    More about Separator
+                  </PrototypeAnchor>
+                </ListItem>
+                <ListItem>
+                  Be careful when adding content from one page to another. You must have{' '}
+                  <strong>import statements for every component</strong> used on a page. When copying and pasting
+                  patterns, include the imports.
+                </ListItem>
+              </UnorderedList>
+            </DisclosureContent>
+          </Disclosure>
+        </Card>
+        <Card>
+          <Heading as="h3" variant="heading30">
+            6. Check Out Patterns
+          </Heading>
+          <Paragraph>Reusable patterns are created from putting many components together for a specific use.</Paragraph>
```

Comment:
I wonder if it would be helpful to link to the Paste patterns docs page here

Reply:
I think this is a good idea, although I'm not sure where I'd put the link. I don't necessarily want them clicking on it first, then coming back and seeing that some of the patterns are "missing". However, I also don't know if they'll return to this page after going to one of the pattern pages. Hmmm... maybe a short link to it at the bottom of this step, like this?
![Screen Shot 2021-02-02 at 2 39 32 PM](https://user-images.githubusercontent.com/77117846/106672160-7b051e00-6564-11eb-8627-9a5ffd297ad9.png)

Reply:
Yeah I love that! Both are good points.

Reply:
Great, thank you!